websites/M/Monos Chinos/dist/metadata.json

```diff
@@ -7,7 +7,7 @@
   "description": {
     "en": "Monos Chinos is a streaming website where you can watch anime with spanish subtitles."
   },
-  "url": "monoschinos.com",
+  "url": ["monoschinos.com", "monoschinos2.com"],
```

Comment:
Why have two URLs if the other one is invalid?

Reply:
ummm, this website uses the first url as redirection to second, I'm going to keep it with the second one